universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -253,6 +257,8 @@ private ProgramDefinition insertProgramWithBlocks(String name) {
   }
 
   private void truncateTables() {
+    versionRepository.getActiveVersion().empty(this);
+    versionRepository.getDraftVersion().empty(this);
```

Comment:
Suggestion (blocking): I'd lean toward truncating version table as well as all the old version would not work, i.e. cannot be revive, after all programs and questions being deleted.

Reply:
I'll give that a try.